packages/vscode-ui5-language-assistant/test/suite/validations/flex-disabled/flex-disabled-spec.ts

```diff
@@ -0,0 +1,54 @@
+import * as vscode from "vscode";
+import { resolve, dirname } from "path";
+import { deactivate } from "../../../../src/extension";
+import { setXMLContent, testDiagnostics, sleep } from "../../test-utils";
+
+const pkgJsonPath = require.resolve(
```

Comment:
why do we use `require.resolve` here? 
- in theory it could also resolve to a folder outside the mono-repo.
- Also it will break if/when we change the extension's name.

I recommend resolving using relative path `__dirname`